app/tests/components/Dashboard/Dashboard.test.tsx

```diff
@@ -0,0 +1,248 @@
+                  closeTimestamp: '2024-09-09T13:49:23.513427-07:00',
+                  openTimestamp: '2022-07-25T00:00:00-07:00',
+                },
+              },
+            ],
+          },
+        };
+      },
+    };
+    componentTestingHelper.loadQuery(payload);
+    componentTestingHelper.renderComponent();
+    expect(screen.getByText('CCBC-010004')).toBeInTheDocument();
+    expect(screen.getByText('submitted')).toBeInTheDocument();
+    expect(screen.getByText('Edit')).toBeInTheDocument();
+    expect(screen.getByTestId('withdraw-btn-test')).toBeInTheDocument();
+  });
+
+  it('Renders a submitted application with the intake closed', () => {
+    const payload = {
+      Query() {
+        return {
+          allApplications: {
+            nodes: [
+              {
+                id: 'WyJhcHBsaWNhdGlvbnMiLDJd',
+                rowId: 2,
+                owner: '4e0ac88c-bf05-49ac-948f-7fd53c7a9fd6',
+                referenceNumber: 1,
+                status: 'submitted',
+                projectName: null,
+                ccbcId: 'CCBC-010005',
+                lastEditedPage: '',
+                intakeByIntakeId: {
+                  ccbcIntakeNumber: 1,
+                  closeTimestamp: '2021-09-09T13:49:23.513427-07:00',
+                  openTimestamp: '2020-07-25T00:00:00-07:00',
+                },
+              },
+            ],
+          },
+        };
+      },
+    };
+    componentTestingHelper.loadQuery(payload);
+    componentTestingHelper.renderComponent();
+    expect(screen.getByText('CCBC-010005')).toBeInTheDocument();
+    expect(screen.getByText('submitted')).toBeInTheDocument();
+    expect(screen.getByText('View')).toBeInTheDocument();
+    expect(screen.queryByTestId('withdraw-btn-test')).toBeNull();
+  });
+
+  it('Renders a withdrawn application', () => {
+    const payload = {
+      Query() {
+        return {
+          allApplications: {
+            nodes: [
+              {
+                id: 'WyJhcHBsaWNhdGlvbnMiLDJd',
+                rowId: 2,
+                owner: '4e0ac88c-bf05-49ac-948f-7fd53c7a9fd6',
+                referenceNumber: 1,
+                status: 'withdrawn',
+                projectName: null,
+                ccbcId: 'CCBC-010005',
+                lastEditedPage: '',
+                intakeByIntakeId: {
+                  ccbcIntakeNumber: 1,
+                  closeTimestamp: '2024-09-09T13:49:23.513427-07:00',
+                  openTimestamp: '2022-07-25T00:00:00-07:00',
+                },
+              },
+            ],
+          },
+        };
+      },
+    };
+    componentTestingHelper.loadQuery(payload);
+    componentTestingHelper.renderComponent();
+    expect(screen.getByText('CCBC-010005')).toBeInTheDocument();
+    expect(screen.getByText('withdrawn')).toBeInTheDocument();
+    expect(screen.getByText('View')).toBeInTheDocument();
+    expect(screen.queryByTestId('withdraw-btn-test')).toBeNull();
+  });
+});
```

Comment:
these are all valuable test cases; what's missing now is db unit tests to confirm the expected permissions for mutability and access are respected. These tests confirm that the user can see whether something should be editable or view only but not whether those applications are actually edit-protected or view-protected. The additional tests should be done at pgtap level.